runner.js

```diff
@@ -6,13 +6,44 @@
 
 const num  = +process.argv.slice(2)[0] || 33;
 
-const fs  = require('fs');
-const evt = require('lib/events');
-const cp  = require('child_process');
-const tp  = require('core/transport');
-const env = Object.assign({DELEGATES: num}, process.env);
+const rb      = require('crypto').randomBytes;
+const fs      = require('fs');
+const cp      = require('child_process');
+const evt     = require('lib/events');
+const Genesis = require('lib/genesis');
+const tp      = require('core/transport');
+const Account = require('core/account');
 
-const kids     = Array.from(Array(num)).map((e, i) => spawnKid(i));
+/**
+ * Initial voting power of the delegate.
+ *
+ * @type {Number}
+ */
+const DELEGATE_BALANCE = 100;
+
+/**
+ * Path to genesis file.
+ *
+ * @type {String}
+ */
+const GENESIS_PATH = `data/${rb(4)}.json`;
+
+const env = Object.assign({DELEGATES: num, GENESIS_PATH}, process.env);
+
+let delegates = [];
```

Comment:
You may want to use Map(<address> => <kid>); Not sure whether it's really needed but could be helpful.
Anyways it will make `const` possible.